js/index.js

```diff
@@ -0,0 +1,56 @@
+//copyright vars
+const today = new Date();
+const thisYear = today.getFullYear();
+const footer = document.querySelector("#footer");
+const copyright = document.createElement("p");
+
+//copyright logo
+const logoCopyright = document.createElement("p");
+logoCopyright.innerHTML = "&#9881";
+logoCopyright.style.display = "inline";
+footer.appendChild(logoCopyright);
+
+copyright.innerHTML = "Cristian Moran" + "- " + thisYear;
+copyright.style.display = "inline";
+footer.appendChild(copyright);
+
+var headings = document.querySelectorAll("h2");
+headings.forEach(function (heading) {
+  heading.style.color = "#000080";
+});
+
+//this is a variable refering to the buttonContainer
+var container = document.querySelector("nav button");
+
+container.addEventListener("click", function (event) {
+  var clicked = event.target.id;
+  if (clicked == "button" || "About") {
+    alert("Welcome to my page!");
+  } else if (clicked == "button" || "Experience") {
+    alert("None lol");
+  } else if (clicked == "button" || "button") {
+    alert("Follow for follow?");
+  }
+});
```

Comment:
Ooh fancy, jumping ahead to the event listeners! This is a clever way to handle a bunch of different things in the same event listener, but usually we try to break them up more, so there's a separate event listener for each user interaction. 

So in this example maybe you'd have 4 different queries for the different nav buttons, as well as 4 different event listeners. Don't worry about it for this PR, just something to think about for later!